java/runtime/src/main/java/org/ray/runtime/raylet/RayletClientImpl.java

```diff
@@ -1,27 +1,25 @@
-package org.ray.spi.impl;
+package org.ray.runtime.raylet;
 
 import com.google.flatbuffers.FlatBufferBuilder;
 import java.nio.ByteBuffer;
 import java.nio.ByteOrder;
 import java.util.ArrayList;
 import java.util.List;
 import java.util.Map;
-
 import org.ray.api.RayObject;
 import org.ray.api.WaitResult;
 import org.ray.api.id.UniqueId;
-import org.ray.core.AbstractRayRuntime;
-import org.ray.core.UniqueIdHelper;
-import org.ray.spi.LocalSchedulerLink;
-import org.ray.spi.model.FunctionArg;
-import org.ray.spi.model.TaskSpec;
-import org.ray.util.ResourceUtil;
-import org.ray.util.logger.RayLog;
-
-/**
- * JNI-based local scheduler link provider.
- */
-public class DefaultLocalSchedulerClient implements LocalSchedulerLink {
+import org.ray.runtime.AbstractRayRuntime;
+import org.ray.runtime.generated.Arg;
+import org.ray.runtime.generated.ResourcePair;
+import org.ray.runtime.generated.TaskInfo;
+import org.ray.runtime.generated.TaskLanguage;
+import org.ray.runtime.task.FunctionArg;
+import org.ray.runtime.task.TaskSpec;
+import org.ray.runtime.util.UniqueIdHelper;
+import org.ray.runtime.util.logger.RayLog;
+
+public class RayletClientImpl implements RayletClient {
```

Comment:
I think  it will be more appropriate for to rename `RayletClientImpl ` to `DefaultRayletClientImpl `.

Reply:
And the `MockRayletClient` can be renamed to `MockRayletClientImpl` also.